src/provider.tsx

```diff
@@ -22,6 +22,7 @@ export const IntercomProvider: React.FC<IntercomProviderProps> = ({
   onUnreadCountChange,
   shouldInitialize = !isSSR,
   apiBase,
+  initializeDelayInMs,
```

Comment:
I would call this just `initializeDelay`, we'll mention the unit in the jsdocs

Reply:
Done in https://github.com/devrnt/react-use-intercom/pull/236/commits/2b733463f2a223babaf1e262063b291c1882bf6a